Sources/OktaAnalytics/AnalyticsStorage.swift

```diff
@@ -152,6 +152,25 @@ class AnalyticsStorage {
         }
     }
 
+    func fetchScenariosAndProperties(expirationPeriod secondsAgo: UInt, completion: @escaping ([Scenario], [ScenarioProperty]) -> Void) {
```

Comment:
If scenario-property is one-to-many relationship why we didn't design model in the way where Scenario owns Property?
Such `[Scenario], [ScenarioProperty]` looks really strange in API contract